espresso/docker-compose.yml

```diff
@@ -176,10 +203,17 @@ services:
       - ../packages/contracts-bedrock/lib/superchain-registry/ops/testdata/monorepo:/config
     command:
       - op-proposer
-      - --l1-eth-rpc=http://l1:8545
-      - --rollup-rpc=http://op-node-sequencer:8545
-      - --game-factory-address=0xDC9a4dba410aaC9D98a848710Aa82601752DBd44
-      - --proposal-interval=10m
+      - --l1-eth-rpc=http://l1:8545                                             # Port from the `l1` service
+      - --rollup-rpc=http://op-node-sequencer:9545                              # Port from the `op-node-sequencer` service
+      # TODO: Fix the address below by deploying the contract.
+      # <https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1210662897601844?focus=true>
+      # We need to specify either
+      #   `l2oo-address` or
+      #   `game-factory-address` and `proposal-interval`.
+      - --game-factory-address=0x80741a37E3644612F0465145C9709a90B6D77Ee3
```

Comment:
Use some constant here as well.

Reply:
It's possible that we won't need this value, or may use a different value after fixing the contract deployment, so I updated the TODO comment to use a const and move this to the `environment` section _after_ fixing the contract.